fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/drawer/Drawer.kt

```diff
@@ -628,124 +628,118 @@ private fun TopDrawer(
         val fullHeight = constraints.maxHeight.toFloat()
         var drawerHeight by remember(fullHeight) { mutableStateOf(fullHeight) }
 
-        //Get exact drawerHeight first.
-        val visible = remember { mutableStateOf(true) }
-        if (visible.value) {
-            Box(
-                modifier = Modifier
-                    .layout { measurable, constraints ->
-                        val placeable = measurable.measure(constraints)
-                        layout(placeable.width, placeable.height) {
-                            visible.value = false
-                            drawerHeight =
-                                placeable.height.toFloat() + dpToPx(DrawerHandleHeightOffset)
-                        }
+        Box(
```

Comment:
Removed this Box. Calculate height in onPlaced modifier & apply alpha  which is set to 0 initally & reset to 1 on height calculation